js/stores/windowStore.js

```diff
@@ -175,6 +177,21 @@ const doAction = (action) => {
       // Since this value is bound we need to notify the control sync
       windowStore.emitChanges()
       return
+    case WindowConstants.WINDOW_SET_FRAME_TAB_ID:
+      windowState = windowState.mergeIn(['frames', FrameStateUtil.getFramePropsIndex(windowState.get('frames'), action.frameProps)], {
+        tabId: action.tabId
```

Comment:
i think `docs/state.md` is supposed to be all app state, regardless of whether it's persistent